### PR TITLE
Positioning and Floating Element: New York Times Clone 

### PR DESCRIPTION
**Pull request by chukwuemeka1234**
In this project, we cloned an article page of the new york times website using HTML and CSS.
Additional information:

- Setup project dependencies like Github actions and Linters via Microverse Linters config
- Defined a well structured DOM with each element properly grouped together using the proper layout masters
- Used the browsers developers tool to properly review the website's development strategies
- Used semantic tags where and others where necessary

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,705 @@
...
+          <ul>
+            <li>Home Page</li>
+            <li>World</li>
+            <li>Coronavirus</li>
+            <li>U.S</li>
+            <li>Politics</li>
+            <li>New York</li>
+            <li>Business</li>
+            <li>Tech</li>
+            <li>Science</li>
```
Comment (Dipeshtwis) [inline index.html]: All the footer element uses an anchor (a) tag, any valid value e.g. "#" is okay for its `href` attribute.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status: ♻️ Changes Required ♻️
## Hi @chukwuemeka1234, I'm Dipesh your reviewer for this PR 👩‍🚀 
This setup looks nice and clean.
Also, I have to appreciate that your website looks exactly similar to the original one and you have done a very good job so far. But in order to approve we have to work on some of the changes mentioned below.

## For Approval
- [x] All the footer section elements should have a link.
- [x] Files not related to the project should not be committed and added to .gitignore (e.g. - .vscode).
- [x] The project must use float as well.

## [Optional]
- [ ] It has a fixed footer (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)).
---------

## Reminder

**"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) now, you have 2 code reviews left for this project."**

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

File: index.html (hunk 1 above)
Comment (chukwuemeka1234) [reply index.html]: Thank you, Sir

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+This is the first html-css project in the Microverse curriculum.
+
+## Built With
+
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](http://127.0.0.1:5502/index.html)
```
Comment (descholar-ceo) [inline README.md]: The live demo link should be from a deployed version, it's not your local address here

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+
+
+## Author
+
+👤 **Ani Victor**
+
+- GitHub: [@chukwuemeka1234](@chukwuemeka1234)
+- Twitter: [@avc_victor](@avc_victor)
+- LinkedIn: [Ani chukwuemeka] (Ani chukwuemeka)
```
Comment (descholar-ceo) [inline README.md]: All of these links, no one is working, all of them are broken, kindly fix them out before you merge this branch on main branch

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @chukwuemeka1234  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, but before you merge to the `main branch`, kindly fix issues that I commented in your readme file

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**